pgql-lang/src/test/java/oracle/pgql/lang/PrettyPrintingTest.java

```diff
@@ -702,4 +682,13 @@ private void checkRoundTrip(String query1) throws PgqlException {
     PgqlStatement iR3 = pgql.parse(query3).getPgqlStatement();
     assertEquals(iR2, iR3);
   }
+
+  @Test
+  public void testCreateSuperGraph() throws Exception {
```

Comment:
Is there a restriction regarding duplicate tables?

Reply:
I improved the existing check now as there were two issues:
 - there were no checks for duplicate element tables between two base graphs or between a base graph and a vertex/edge table (note: if users use ALL ELEMENT TABLES [EXCEPT], the parser cannot do the duplicate checking so it's left to the PGQL implementation to do so).
 - in case of ALL ELEMENT TABLES [EXCEPT], the parser would still complain about unresolved references to vertex tables inside EDGE TABLES, but since we do not know all the vertex tables in such case, the parser should not generate any errors for vertex tables that are not declared

https://github.com/oracle/pgql-lang/pull/114/commits/cb44ef989a18bdbc3b2d138e802b7766a789dfe8